rfd/0043-kubeaccess-multiparty.md

```diff
@@ -144,18 +144,38 @@ If tap is made after the alert, the follwing message is shown:
 Teleport >> MFA tap received.
 ```
 
+#### Broadcast messages
```

Comment:
@xacrimon changes as discussed:

1. Define all 4 nouns (participant, observer, moderator, and peer) at the top
2. Change “launching session” to “connecting to $hostname using $protocol” for clarity
3. If moderator leaves (either voluntarily or failed MFA check), participants should see “Session terminated: participant requirements not met”
4. Provide a --participant-req flag that, if set, prints participant requirements after `Session paused, waiting for additional participants...`